public/locales/en/translation.json

```diff
@@ -218,6 +218,25 @@
           }
         }
       },
+      "broadcasts": {
+        "title": "Broadcasts",
+        "columns": {
+          "success": "Success",
+          "start": "Started",
+          "fail": "Failed",
+          "create": "Created",
+          "update": "Updated",
+          "delete": "Deleted"
+        },
+        "rows": {
+          "instance": "Instance",
+          "directory": "Directory",
+          "workflow": "Workflow",
+          "instance-variable": "Instance Variable",
```

Comment:
I think we want the row keys to be in lowerCamelCase as well?

Reply:
fixed